app-tracking-protection/vpn-impl/src/main/res/values/strings-vpn.xml

```diff
@@ -220,13 +220,18 @@
         <item quantity="one">App</item>
         <item quantity="other">Apps</item>
     </plurals>
-    <string name="atp_ActivityEnabledLabel">This feature is in beta. <annotation type="app_settings_link">Having problems with an app?</annotation></string>
+    <string name="atp_ActivityEnabledLabel">This feature is in beta. Notice a problem with an app? <annotation type="app_settings_link">Please report an issue.</annotation></string>
```

Comment:
FYI this is never going to pass lint unless you translate these or mark them as `translatable=false`. Also the branch name will not trigger the Smartling automation (if we ever merge it...). I would suggest to mark all of these as `translatable=false` and as a follow-up remove that tag once the automation is ready.

Reply:
@marcosholgado because of the annotation? If so, the lint rule should know and not flag the string.

Reply:
Nope, because all the strings that are not inside donottranslate.xml or marked as `translatable=false` are expected to be translated so lint will (and it actually is) fail.

Reply:
Spoke offline, all clear. @aitorvs will deal with it.